jdi-light/src/main/java/com/epam/jdi/light/asserts/generic/table/BaseTableAssert.java

```diff
@@ -23,117 +23,150 @@ protected T table() {
         element.refresh();
         return element;
     }
+    
     /**
      * Check that the table is empty
      */
     @JDIAction("Assert is '{name}' empty")
-    public A empty() {
-        jdiAssert(table().isEmpty() ? "is empty" : "is not empty", Matchers.is("is empty"));
+    public A empty(String... messages) {
+        jdiAssert(table().isEmpty() ? "is empty" : "is not empty", Matchers.is("is empty"), messages);
         return (A) this;
     }
-
+    
     /**
      * Check that the table isn't empty
      */
     @JDIAction("Assert is '{name}' not empty")
-    public A notEmpty() {
-        jdiAssert(table().isEmpty() ? "is empty" : "is not empty", Matchers.is("is not empty"));
+    public A notEmpty(String... messages) {
+        jdiAssert(table().isEmpty() ? "is empty" : "is not empty", Matchers.is("is not empty"), messages);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5133577)